bgpd/bgp_route.c

```diff
@@ -8867,8 +8867,13 @@ void route_vty_out_detail(struct vty *vty, struct bgp *bgp,
 		} else {
 			if (nexthop_vrfid == VRF_UNKNOWN)
 				vty_out(vty, " vrf ?");
```

Comment:
maybe replace it to `Unknown` like everywhere else?

Reply:
sounds like another PR at a later date to make things consistently fun.